### PR TITLE
Add Lookup Table resource

### PR DESCRIPTION
**Pull request by sumovishal**
Docs will be added in a separate PR.

### CODE REVIEW

File: sumologic/resource_sumologic_lookup_table.go [hunk 1]
```diff
@@ -119,7 +122,7 @@ func resourceSumologicLookupTableRead(d *schema.ResourceData, meta interface{})
 
 	id := d.Id()
 	lookupTable, err := c.GetLookupTable(id)
-	fmt.Printf("##DEBUG## read lookup: %+v\n", lookupTable)
+	log.Printf("##DEBUG## read lookup: %+v\n", lookupTable)
```
Comment (sumovishal) [inline sumologic/resource_sumologic_lookup_table.go:125]: You can remove these and other print statements I added. I added them for debugging purpose.

File: sumologic/resource_sumologic_lookup_table.go [hunk 2]
```diff
@@ -131,7 +134,9 @@ func resourceSumologicLookupTableRead(d *schema.ResourceData, meta interface{})
 	}
 
 	d.Set("name", lookupTable.Name)
-	d.Set("fields", lookupTable.Fields)
+	if err := d.Set("fields", fieldsToList(lookupTable.Fields)); err != nil {
```
Comment (sumovishal) [inline sumologic/resource_sumologic_lookup_table.go:137]: We need to fix codegen as it doesn't handle this correctly.

**Review by sumovishal — COMMENTED** [review]
lgtm, you can remove the print statements.

File: sumologic/resource_sumologic_lookup_table.go (hunk 1 above)
Comment (seansain) [reply sumologic/resource_sumologic_lookup_table.go:125]: they only come if someone sets `TF_LOG` to `debug`. We can keep them for further debugging.

**Pull request merged**